libs/ui/jest.config.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
   testEnvironment: 'jsdom',
   setupFilesAfterEnv: ['<rootDir>/src/setupTests.ts'],
   coveragePathIgnorePatterns: [
+    'src/reports/index.ts',
```

Comment:
weird - is coverage complaining about an index file with only `export` statements?

Reply:
Yeah :(